modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i

```diff
@@ -226,10 +237,12 @@ velocity_interp_method = 'rc'
     rho_d = ${rho_d}
     particle_diameter = ${dp}
     linear_coef_name = 'Darcy_coefficient'
+    outputs = 'out'
+    output_properties = 'vel_slip_x'
```

Comment:
noice

Reply:
haha I was excited to use your new capability!